### PR TITLE
Proposal for retrieving and observing topics via headers

### PR DESCRIPTION
**Pull request by jkarlin**
As discussed in #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,19 @@ The topics will be inferred by the browser. The browser will leverage a classifi
...
+    * The request header can be sent along with fetch requests via specifying an option: `fetch(<url>, {browsingTopics: true})`.
+    * Redirects will be followed, and the topics sent in the redirect request will be specific to the redirect url.
+    * The request header will not modify state for the caller unless there is a corresponding response header. That is, the topic of the page won't be considered observed, nor will it affect the user's topic calculation for the next epoch. 
+    * The response header will only be honored if the corresponding request included the topics header (or would have included the header if it wasn't empty).
+    * The registrable domain used for topic observation is that of the url of the request.
+    * Example request header: `Sec-Browsing-Topics: 123;model=1;taxonomy=1;version=2, 2;model=1;taxonomy=1;version=2`
+        * This example has two topics, 123 and 2, along with their version information.
+    * Example response header: `Observe-Browsing-Topics: 1`
```
Comment (xyaoinum) [inline README.md:73]: (From @domenic) Should this be "?1", so that it's a structured field https://www.rfc-editor.org/rfc/rfc8941.html#section-3.3.6-4

Comment (michaelkleber) [reply README.md:73]: +1

Comment (jkarlin) [reply README.md]: Whoops, thanks! Done.

**Comment by zhengweiwithoutthei** [comment]
This is exciting! Thanks. Two questions:
1. Is there a support for Xhr API? 
2. What Chrome version this change will be included?

**Comment by jkarlin** [comment]
Yes, it makes sense to add XHR if we have fetch. I'll clarify. We're targeting M108 for these changes, but no guarantees.

**Comment by domenic** [comment]
Actually, there's general cross-browser agreement to not add any new capabilities to XHR...

**Comment by zhengweiwithoutthei** [comment]
Will there be a support for WebBundle? Repeating myself https://github.com/patcg-individual-drafts/topics/issues/7#issuecomment-1042475440 :)

**Pull request merged**